src/System.Net.Http/src/System/Net/Http/HttpRequestException.cs

```diff
@@ -6,7 +6,6 @@
 
 namespace System.Net.Http
 {
-    [Serializable]
     [SuppressMessage("Microsoft.Serialization", "CA2229")]
```

Comment:
> But if we're removing the Serializable attribute, then the type shouldn't be implementing ISerializable anymore

ISerializable is part of the type's signature/shape whereas [Serializable] is treated as an implementation detail.  We can remove [Serializable], but its ISerializable definition remains as its part of the type's APIs / standard.  The implementation will just throw.